app/src/test/java/org/fossasia/susi/ai/helper/DateTimeHelperTest.kt

```diff
@@ -0,0 +1,17 @@
+package org.fossasia.susi.ai.helper
+
+import org.junit.Assert
+import org.junit.Test
+
+class DateTimeHelperTest {
+    @Test
+    fun getDateTest() {
+        Assert.assertEquals(" Jan 30, 2020", DateTimeHelper.getDate("2020-01-30T12:18:37.262Z"))
+    }
+
+    @Test
+    fun formatDateTest() {
+        val countries = arrayOf("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Aug", "Nov", "Dec")
+        Assert.assertEquals("30 Jan, 2020", DateTimeHelper.formatDate("2020-01-30T12:18:37.262Z", countries))
```

Comment:
Add  case for null timestamp as well

Reply:
@iamareebjamal added null checks as well as tests for null and empty values. One question though, I tried adding test for one remaining function ie getTime() but every time CI build is failing. That's because i have to hardcode expectation for input, but expectations would be different depending upon different timezone(github vs my), so according to me it's not possible... or is it??

Reply:
See tests for  open-event-organizer app

Reply:
Thanks... that worked, implemented tests for different timezones